docs/qa/CheckLists/SettingsPage.md

```diff
@@ -0,0 +1,40 @@
+| 3  | Pressing "Show welcome page" radio button enables/disables appearance of "welcome" page (design page 1)                                                    |             |                                                                                                                                         |
+| 4  | Pressing "Clear Recycle bin every" button opens up a bottom sheet with 4 options, it can be checked on (design page 26.1)                                  |             |                                                                                                                                         |
+| 5  | All 4 options of "Clear Recycle bin every" work as intended                                                                                                |             |                                                                                                                                         |
+| 6  | Pressing "Proceed to Recycle bin" button leads to recycle bin page (design page 28)                                                                        |             |                                                                                                                                         |
+| 7  | Pressing "Help" button leads to help page (design page 27)                                                                                                 |             |                                                                                                                                         |
+| 8  | Pressing "Licenses" button leads to somewhere                                                                                                              |             |                                                                                                                                         |
+|    | 27 Help                                                                                                                                                    |             |                                                                                                                                         |
+| 9  | Check all UI texts/elements on (design page 27)                                                                                                            |             | Design page 27                                                                                                                          |
+| 10 | When page opens up all questions are hidden behind headers                                                                                                 |             | Will state of open/close saved between sessions? Example, user opens up F.A.Q and there already opened questions that he open up before |
```

Comment:
open**s**

Reply:
Changed to "When page is opened all questions are hidden behind headers"